src/less/forms.less

```diff
@@ -800,6 +801,97 @@ input[type='checkbox']:focus {
   }
 }
 
+// add other attr to input to identify it in css file, similar to checked
+
+.rhinoslidingradio__input {
+  &:checked + .rhinoslidingradio__label::after {
+    transform-origin: center center;
+    transform: scale(1.5, 1.5);
+    transition: all 0.25s @global-trans-timing-cubic;
```

Comment:
I took some time to review our use of transitions and noticed we've got the following values being used:

0.25s
250ms
0.3s
0.5s
0.1s

This isn't integral to this specific PR but what do you think about creating some variables for `transition-speed-slow` `transition-speed-normal` and `transition-speed-fast`

To try and reign all of these in to some shared standards? Would love to hear your thoughts.